bot/cogs/snakes.py

```diff
@@ -29,8 +35,31 @@ def __init__(self, bot: AutoShardedBot):
         :return: A dict containing information on a snake
         """
 
+    async def get_snek_qwant_json(self, snake_name):
+        """
+        Gets the json from Unsplash for a given snake query
+        :param snake_name: name of the snake
+        :return: the full JSON from the search API
+        """
+        url = "https://api.unsplash.com/search/photos?client_id" \
+              "={0}&query={1}".format(os.environ.get("UNSPLASH_CLIENT_ID"), snake_name)
+        async with aiohttp.ClientSession() as session:
+            async with session.get(url) as response:
+                response = await response.read()
+                return json.loads(response.decode("utf-8"))
+
+    async def get_snek_image(self, name):
+        """
+        Gets the URL of a snake image
+        :param name: name of snake
+        :return: image url
+        """
+        json_response = await self.get_snek_qwant_json(name)
+        rand = random.randint(0, 9)  # prevents returning the same image every time
+        return json_response['results'][rand]['urls']['small']
+
     @command()
-    async def get(self, ctx: Context, name: str = None):
+    async def get(self, ctx: Context, name: str = "python"):
```

Comment:
No, this should default to `None`. The task requires a random snake  if the parameter is left out.

Reply:
yep - just noticed this - thanks!